addon_service/citation_operation_invocation/models.py

```diff
@@ -0,0 +1,109 @@
+import traceback
+
+import jsonschema
+from django.core.exceptions import ValidationError
+from django.db import models
+
+from addon_service.common.base_model import AddonsServiceBaseModel
+from addon_service.common.invocation_status import InvocationStatus
+from addon_service.common.known_imps import AddonImpNumbers
+from addon_service.common.validators import validate_invocation_status
+from addon_service.models import AddonOperationModel
+from addon_toolkit import AddonImp
+from addon_toolkit.interfaces.citation import CitationConfig
+
+
+class CitationOperationInvocation(AddonsServiceBaseModel):
+    int_invocation_status = models.IntegerField(
+        validators=[validate_invocation_status],
+        default=InvocationStatus.STARTING.value,
+    )
+    operation_identifier = models.TextField()  # TODO: validator
+    operation_kwargs = models.JSONField(default=dict, blank=True)
+    thru_addon = models.ForeignKey(
+        "ConfiguredCitationAddon", null=True, blank=True, on_delete=models.CASCADE
+    )
+    thru_account = models.ForeignKey(
+        "AuthorizedCitationAccount", on_delete=models.CASCADE
+    )
+    by_user = models.ForeignKey("UserReference", on_delete=models.CASCADE)
```

Comment:
Shouldn't citation operation invocation be removed?

Reply:
Removed it